mage_ai/frontend/pages/pipelines/[pipeline]/runs/[run]/index.tsx

```diff
@@ -65,7 +72,9 @@ function PipelineBlockRuns({
 
   const { data: dataPipelineRun } = api.pipeline_runs.detail(
     pipelineRunProp.id,
-    {},
+    {
+      exclude_block_runs: true,
```

Comment:
Added this query param since we need to fetch the pipeline run, but we no longer need to include the block_runs in that payload. We fetch the block runs with a separate block runs request which already handles the `_limit` and `_offset` params.